### ISSUE TITLE
DQN pong example doesn't work off the shelf

### ISSUE THREAD

**Issue by thisiscam** [coax-dev/coax#37]
**Describe the bug**

Running the DQN example on pong generates the following error when generating a gif:
```python
  File ".../lib/python3.9/site-packages/coax/utils/_misc.py", line 475, in generate_gif
    assert env.render_mode == 'rgb_array', "env.render_mode must be 'rgb_array'"
```

This is likely due to some recent updates to gym. Currently, on `gym==0.26.2` I observe the following:
```python
import gym
env = gym.make('PongNoFrameskip-v4', render_mode="rgb_array")
print(env.render_mode) # prints None
```

**Comment by KristianHolsheimer** [comment]
Thanks for reporting this @thisiscam and thanks so much for digging into this @benjaminbeilharz 

Would you mind filing a bug upstream in https://github.com/openai/gym ?

I'll review your PR now.

**Comment by benjaminbeilharz** [comment]
@KristianHolsheimer I have filed an issue here: https://github.com/openai/gym/issues/3166

Thanks for your work on coax :)

**Comment by KristianHolsheimer** [comment]
Fixed by #38
